scripts/create-sdcard-image-from-scratch.sh

```diff
@@ -1,49 +1,78 @@
 #!/bin/bash
```

Comment:
I can't tell where jupyter clickable image widget is installed.  Seems like this would break the main data_collection notebook.

We'll need to add that here.

Is this bottlenecked by merging the non-typescript version to master of jupyter widget?

Reply:
Oh, good point.
I noticed it was missing the jupyter_clickable_image_widget and put it back when I tested, but forgot to put that to this branch.
Let me add the lines to the install script.
It is not that bottlenecked, we can for now just clone my fork of the repo.

Reply:
I'll just merge your PR based on no-typescript to master.  Then we only have 1 to worry about

Reply:
OK see.  

https://github.com/jaybdub/jupyter_clickable_image_widget/pull/5

I've added a release/tag.  Base off that for stability.

https://github.com/jaybdub/jupyter_clickable_image_widget/releases/tag/v0.1